code/modules/food_and_drinks/food/snacks.dm

```diff
@@ -130,14 +132,19 @@
 			istype(W, /obj/item/weapon/hatchet) \
 		)
 		inaccurate = 1
-	else if(W.w_class <= WEIGHT_CLASS_SMALL && istype(src,/obj/item/weapon/reagent_containers/food/snacks/sliceable))
+	if(total_w_class > MAX_WEIGHT_CLASS)
```

Comment:
I don't think we want to be able to stuff things in nonsliceable food.

Reply:
I blame my lack of sleep, one moment

Reply:
@DarkPyrolord You replaced the wrong `if()` statement :stuck_out_tongue: 
Leave this one as it was, 
`else if(W.w_class <= WEIGHT_CLASS_SMALL && istype(src,/obj/item/weapon/reagent_containers/food/snacks/sliceable))`